java-checks-test-sources/src/main/java/symbolicexecution/checks/S2755_XxeProcessingCheck_DocumentBuilderFactory.java

```diff
@@ -61,6 +70,23 @@ DocumentBuilderFactory set_external_general_false() throws ParserConfigurationEx
     return factory;
   }
 
+  void secure_with_no_op_entity_resolver(InputStream is) throws Exception {
+    DocumentBuilderFactory df = DocumentBuilderFactory.newInstance();
+    DocumentBuilder builder1 = df.newDocumentBuilder(); // Compliant thanks to "builder.setEntityResolver"
+    builder1.setEntityResolver((publicId, systemId) -> new InputSource(new StringReader("")));
+    Document doc1 = builder1.parse(is);
+    DocumentBuilder builder2 = df.newDocumentBuilder(); // Noncompliant
+    Document doc2 = builder2.parse(is);
+  }
+
+  void insecure_with_null_entity_resolver(InputStream is) throws Exception {
+    DocumentBuilderFactory df = DocumentBuilderFactory.newInstance();
+    DocumentBuilder builder = df.newDocumentBuilder(); // Noncompliant
+    builder.setEntityResolver((publicId, systemId) -> new InputSource(new StringReader("")));
+    builder.setEntityResolver(null);
```

Comment:
I've seen Eric suggested secondary location, for this case. Have you decided to do it later or not to do at all?